vpr/SRC/route/route_timing.c

```diff
@@ -17,14 +18,31 @@
 #include "net_delay.h"
 #include "stats.h"
 #include "ReadOptions.h"
-
-
+ 
 using namespace std;
 
+/***************** Iterative connection based rerouting **********************/
 

```

Comment:
Include high-level description of how connection-based rerouting works and the major functions that are involved.